docs/UserGuide.md

```diff
@@ -103,24 +91,35 @@ Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG] [m/MATRICULATION_NU
 **Tip:** A person can have any number of tags (including 0)
 </box>
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 m/A1234567Z`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal m/A1234567Z`
+Example:
+`add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 m/A1234567Z s/S1 r/R2`
+
+>A success message for the above command will be shown as followed, with the new person in the contact list:
+>
+>
+>![add success message](images/success_images/add_success.png)
```

Comment:
nice adding images on what a successful command would look like as users will be visualise what the UI should look like when their inputs are successful